sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -45,6 +54,37 @@
       @pageChanged="pageChanged"
     />
   </student-page-container>
+  <v-row v-if="showNav">
```

Comment:
shouldn't  these btns inside "student-page-container"?

Reply:
are these btns working? I see same btns in `sources/packages/web/src/components/common/StudentApplication.vue`

Reply:
Yeah, actually we don't need those buttons here. Thanks!